assets/dropdown-menu.js

```diff
@@ -0,0 +1,46 @@
+const dropdownInput = $('.dropdown-input');
+const dropdownBtn = $('.dropbtn');
+const dropdownContent = $('.dropdown-content');
+const dropdownOptions = dropdownContent?.querySelectorAll('li');
+
+function showDropDownMenu(customContent) {
+  if(customContent) {
+      customContent.classList.toggle('show');
+      dropdownInput.classList.toggle('black-border');
+  } else if (dropdownContent) {
+      dropdownContent.classList.toggle('show');
+      dropdownInput.classList.toggle('black-border');
+  }
+}
+
+const hideDropDownMenu = (event) => {
+  if (!event.target.matches('.dropdown-input, .dropdown-input *')) {
+    dropdownContent?.classList.remove('show');
+    dropdownInput?.classList.remove('black-border');
+  }
+};
+
+window.addEventListener('click', (event) => hideDropDownMenu(event));
+
+function selectOption(e) {
+  const selectedOption = e.target;
+
+  dropdownOptions.forEach((option) => {
+    option.classList.remove('selected');
+  });
+
+  selectedOption.classList.add('selected');
+  dropdownBtn.firstElementChild.textContent = selectedOption.textContent;
+}
+
+if(dropdownOptions) {
+  dropdownOptions.forEach((option) => {
+    option.addEventListener('click', selectOption);
+
+    window.addEventListener('DOMContentLoaded', (event) => {
+      if (option.classList.contains('selected')) {
+        dropdownBtn.firstElementChild.textContent = option.textContent;
+      }
+    });
+  });
+}
```

Comment:
can you call hire the function of hideDropDownMenu() when the user select in option the dropdown should be closed 
check on the collections page the dropdown menu should be closed on the option select

Reply:
in collections page when you select item from the list the page refreshed because we have anchor tags

Reply:
there is no need for hiding the dropdown

Reply:
I already implemented this on the dropdown-menu.js when we don't have an anchor tag